packages/flutter/test/widgets/image_test.dart

```diff
@@ -318,6 +318,22 @@ void main() {
     expect(renderer.color, const Color(0xFF00FF00));
     expect(renderer.colorBlendMode, BlendMode.clear);
   });
+
+  testWidgets('Prefetch', (WidgetTester tester) async {
+    final TestImageProvider provider = new TestImageProvider();
+    Future<Null> prefetch;
+    await tester.pumpWidget(
+      new Builder(
+        builder: (BuildContext context) {
+          prefetch = prefetchImage(provider, context);
+          return new Container();
+        }
+      )
+    );
+    provider.complete();
+    await prefetch;
+    expect(provider._configuration, isNotNull);
+  });
 }
 
 class TestImageProvider extends ImageProvider<TestImageProvider> {
```

Comment:
Can you rename TestImageProvider._configuration to TestImageProvider._lastResolvedConfiguration? The test you added confused me for a while until I realised why the configuration was going to be set.